administrator/components/com_associations/models/fields/menuparent.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_menus
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('JPATH_BASE') or die;
+
+JFormHelper::loadFieldClass('list');
```

Comment:
No idea, but searching:
https://github.com/joomla-projects/gsoc16_improved-multi-lingual/search?utf8=%E2%9C%93&q=JFormFieldList
confirms that it's needed :stuck_out_tongue: